cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/HBaseTableUtil.java

```diff
@@ -93,10 +94,14 @@
   public static final String CFG_HBASE_TABLE_COMPRESSION = "hbase.table.compression.default";
 
 
-  protected CConfiguration cConf;
+  protected String tablePrefix;
 
   public void setCConf(CConfiguration cConf) {
-    this.cConf = cConf;
+    this.tablePrefix = cConf.get(Constants.Dataset.TABLE_PREFIX);
+  }
+
+  protected boolean isCDAPTable(String hTableName) {
```

Comment:
It should only be moved to subclasses if we can operate on ``TableId``. 

I'm only afraid that we may end up using this for non-fully qualified table names (without namespace, perhaps?)